doc-content/enterprise-only/integration/dev-studio-setting-drools-runtime-proc.adoc

```diff
@@ -0,0 +1,23 @@
+[id='dev-studio-setting-drools-runtime-proc']
+= Setting Drools Runtime
+
+In order to create Drools' projects in Red Hat Developer Studio, it is necessary to set up the runtime.
+
+A runtime is a collection of JAR files that represents a specific release of the software and provides libraries needed for compilation and running of your business assets.
+
+.Prerequisite
+Red Hat Developer Studio is installed.
+
+.Procedure
+. Download the Drools Engine:
+.. Log in to the https://access.redhat.com[Red Hat Customer Portal].
+.. Click *DOWNLOADS* at the top of the page.
+.. On the *Product Downloads* page that opens, navigate to the JBOSS DEVELOPMENT AND MANAGEMENT section, and click *Red Hat Decision Manager*.
+.. On the *Software Downloads* page, download *Red Hat Decision Manager 7.1.1 Add-Ons* (`rhdm-7.1.1-add-ons.zip`).
+.. Extract `rhdm-7.1.1-add-ons.zip` and then extract the Drools runtime JAR files located in `rhdm-7.1.1-add-ons/rhdm-7.1-decision-engine.zip`.
+. From the Red Hat Developer Studio menu, click *Window* -> *Preferences*.
+. Click *Drools* -> *Installed Drools Runtimes*, and then click *Add*.
+. In the name field, enter a name for the new runtime.
+. Click *Browse* and navigate to the directory where you extracted the runtime files. Click *OK* to register the selected runtime in Red Hat Developer Studio.
+. To set the runtime as the default Drools runtime, select the check box beside the runtime name.
+. Click *Apply and Close*. If you have existing projects, a dialog box will indicate that you have to restart Red Hat Developer Studio to update the runtime.
```

Comment:
If you have existing projects, a dialog box appears instructing you to restart... (or something like that).
Always be suspicious of 'will'. It's not that you can never you use it but chances are that it is in a passive construction and we need to avoid that.

Reply:
How about "If you have existing projects, you will be prompted to restart Red Hat Developer Studio to update the run time environment." ?

Reply:
You gotta lose the will :-) you are prompted :-)

Reply:
ahhh I will not do that again.

Reply:
Adopted phrasing